analyzers/src/SonarAnalyzer.CSharp/LiveVariableAnalysis/SonarCSharpLiveVariableAnalysis.cs

```diff
@@ -68,21 +68,21 @@ internal static bool IsLocalScoped(ISymbol symbol, ISymbol declaration)
 
         protected override State ProcessBlock(Block block)
         {
-            var ret = new SonarState(originalDeclaration, semanticModel);
+            var ret = new SonarState(this, semanticModel);
             ret.ProcessBlock(block);
             return ret;
         }
 
         private class SonarState : State
         {
-            private readonly ISymbol originalDeclaration;
+            private readonly SonarCSharpLiveVariableAnalysis owner;
```

Comment:
I don't really think that the name could match the type. It's like 2 words from different categories